monitor_mqtt/rules/sample_rule.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I would delete this file. This is going to install a rule in everyone's stackstorm instance to send you an email.

Reply:
deleted